src/app/pages/homepage/homepage.component.html

```diff
@@ -12,7 +12,7 @@
       <li><a>Score: 713</a></li>
       <li><a>user.email</a></li>
       <li><a><i class="fas fa-envelope"> Messages</i></a></li>
-      <li><a><i class="fas fa-cogs"></i></a></li>
+      <li><a (click)="quit()"><i class="fas fa-cogs"></i></a></li>
```

Comment:
Place "click" event in "<li click="quit()">" tag, because user shuld click on list item but not only on text.